### PR TITLE
Add Kotlin to all polyglot pages

### PR DESCRIPTION
**Pull request by mlvandijk**
Added Kotlin to all polyglot pages
- [x] api.md
- [x] configuration.md 
- [x] cucumber-expressions.md
- [x] reporting.md
- [x] step-definitions.md
- [x] anti-patterns.md
- [x] browser-automation.md

### CODE REVIEW

File: content/community/contributing-to-documentation.md [hunk 1]
```diff
@@ -123,20 +94,24 @@ A language select will be displayed if the page specifies the following in the f
...
  ```
 
 The following languages are currently supported:
 
 * Java
 * JavaScript
 * Ruby
+* Kotlin (optional)
 * .Net (optional; some pages only)
+* Kotlin (optional)
```
Comment (luke-hill) [inline content/community/contributing-to-documentation.md:110]: Is there a reason this is mentioned twice above and here?

Comment (mlvandijk) [reply content/community/contributing-to-documentation.md:110]: One is an example of the polyglot front matter, and one is a list of the currently supported languages. Is that what you mean?

**Comment by olleolleolle** [comment]
This is beginning to look _complete_. That's cool!

**Comment by mlvandijk** [comment]
@olleolleolle Afaik I added Kotlin to all polyglot pages and added a Kotlin translation for all code examples. Please let me know if you think there are any missing?
Also, a friend reviewed the Kotlin code and I did some fixes based on his review ;)
So imho this PR is good to go!

**Comment by olleolleolle** [comment]
I think we are ready for a merge.

**Review by olleolleolle — APPROVED** [review]
LGTM

**Comment by mlvandijk** [comment]
Thanks @olleolleolle

File: content/community/contributing-to-documentation.md (hunk 1 above)
Comment (mpkorstanje) [reply content/community/contributing-to-documentation.md:110]: I think he means, why does kotlin occur four times in two lists? Shouldn't it occur only twice in two lists?

File: content/guides/browser-automation.md [hunk 2]
```diff
@@ -77,6 +77,48 @@ public class ExampleSteps {
...
+import cucumber.api.Scenario
+import cucumber.api.java8.En
+import org.openqa.selenium.By
+import org.openqa.selenium.WebDriver
+import org.openqa.selenium.WebElement
+import org.openqa.selenium.support.ui.WebDriverWait
+
+class ExampleSteps: En {
+
+    lateinit var driver: WebDriver
```
Comment (mpkorstanje) [inline content/guides/browser-automation.md:92]: Why `lateinit`?

File: content/community/contributing-to-documentation.md (hunk 1 above)
Comment (mlvandijk) [reply content/community/contributing-to-documentation.md:110]: Yes, found this too. Fixed it ;)

File: content/guides/browser-automation.md (hunk 2 above)
Comment (mlvandijk) [reply content/guides/browser-automation.md]: Hmm, usually I'd autowire it, in which case I use lateinit var. Maybe not needed here.

Comment (mlvandijk) [reply content/guides/browser-automation.md:92]: `lateinit `prevents having to initialize here, and making WebDriver nullable.
Personally, I prefer `lateinit var driver: WebDriver` to `private var driver: WebDriver? = null`

Comment (mpkorstanje) [reply content/guides/browser-automation.md:92]: But shouldn't the webdriver be injected then? It is never instantiated.

**Pull request merged**